.github/workflows/wails.yml

```diff
@@ -200,7 +200,7 @@ jobs:
           mv ./build/out/${{ env.PACKAGE_NAME }}.dmg ./build/bin/
 
       - name: Notarize the DMG image
-        if: runner.os == 'macOS' && inputs.build-release
+        if: runner.os == 'macOS'
```

Comment:
was this intentional?

Reply:
Yes, we have to notarize this everytime, otherwise only signing wouldn't work (unless you already opened the app before) And we have a limit of 70 notarizations per day (with our apple dev account) so this should be good atleast for now

Reply:
@im-adithya but people only run the release builds, don't they?

Reply:
Yeah but I thought if someone of us is testing for the first time, they might see this issue. But on second thoughts it might hit the 70 limit someday and might create problem during release, I'll add this back 👍